src/include/platform/internal/GenericPlatformManagerImpl_FreeRTOS.ipp

```diff
@@ -294,19 +405,6 @@ void GenericPlatformManagerImpl_FreeRTOS<ImplClass>::PostEventFromISR(const Chip
     }
 }
 
-template <class ImplClass>
```

Comment:
Why are these being moved?

Reply:
I've removed the reordering changes in this file.